### PR TITLE
fix error on service estimations

### PR DESCRIPTION
**Pull request by lieblichaaron**
## ℹ️ Change description:
Nothing here

<details><summary><sub>templates<sub></summary>
<p>
  
<sub>[templates: [bug](?expand=1&template=bug.md), [feature](?expand=1&template=feature.md)]</sub>

</p>
</details>

### CODE REVIEW

File: examples/client/Locomotion/src/context/newRideContext/index.tsx [hunk 1]
```diff
@@ -219,13 +219,19 @@ const RidePageContextProvider = ({ children }: {
       const formattedEstimations = formatEstimations(services, estimations, tags);
       setChosenService(formattedEstimations.find((e: any) => e.eta));
       setServiceEstimations(formattedEstimations);
+      setIsLoading(false);
     } catch (e) {
+      setIsLoading(false);
       setServiceRequestFailed(true);
       setIsReadyForSubmit(false);
-    } finally {
```
Comment (OmerGery) [inline examples/client/Locomotion/src/context/newRideContext/index.tsx:225]: isnt finally doing the same thing?

**Review by OmerGery — APPROVED** [review]

File: examples/client/Locomotion/src/context/newRideContext/index.tsx [hunk 2]
```diff
@@ -219,13 +219,19 @@ const RidePageContextProvider = ({ children }: {
       const formattedEstimations = formatEstimations(services, estimations, tags);
       setChosenService(formattedEstimations.find((e: any) => e.eta));
       setServiceEstimations(formattedEstimations);
+      setIsLoading(false);
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/context/newRideContext/index.tsx]: Finally should do the same

File: examples/client/Locomotion/src/context/newRideContext/index.tsx [hunk 3]
```diff
@@ -219,13 +219,19 @@ const RidePageContextProvider = ({ children }: {
...
     } catch (e) {
+      setIsLoading(false);
       setServiceRequestFailed(true);
       setIsReadyForSubmit(false);
-    } finally {
-      setIsLoading(false);
     }
   };
+  useEffect(() => {
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/context/newRideContext/index.tsx]: shouldn't be related to the ride page context, please handle it in the component

File: examples/client/Locomotion/src/context/newRideContext/index.tsx [hunk 4]
```diff
@@ -219,13 +219,19 @@ const RidePageContextProvider = ({ children }: {
...
+    if (serviceRequestFailed) {
+      changeBsPage(BS_PAGES.ADDRESS_SELECTOR);
+      stopRequestInterval();
+    }
+  }, [serviceRequestFailed]);
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/context/newRideContext/index.tsx]: I don't like that we have specific variable for the popup, It should be a popup name

**Review by avivbarniv — CHANGES_REQUESTED** [review]
/

**Review by avivbarniv — APPROVED** [review]
Add also type of the popup, u can see notes/payment as a reference

**Pull request merged**